react-app/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-app",
+  "version": "1.0.0",
+  "description": "React App using Babel, Webpack and SASS",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf ./dist/*",
+    "build": "npm run clean && webpack -p",
+    "start": "npm run clean && webpack-dev-server --hot"
+  },
+  "keywords": [
+    "React"
+  ],
+  "author": "Akshay Pawar",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.11",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
You are using the beta version of this npm package which is very dangerous. I would suggest you go with last stable version.